social_core/backends/itembase.py

```diff
@@ -21,7 +21,7 @@ class ItembaseOAuth2(BaseOAuth2):
         ("token_type", "token_type"),
         ("refresh_token", "refresh_token"),
         ("expires_in", "expires_in"),  # seconds to expiration
-        ("expires", "expires"),  # expiration timestamp in UTC
+        ("expires_on", "expires_on"),  # expiration timestamp in UTC
```

Comment:
This defines attribute mapping - the first element defines name in the response, it should not be changed.